config/database.yml

```
@@ -15,3 +15,8 @@ development:
 test:
   <<: *default
   database: <%= ENV.fetch("DB_DATABASE", "register_trainee_teacher_data_test#{ENV['TEST_ENV_NUMBER']}") %>
+
+production:
+  <<: *default
+  variables:
+    statement_timeout: <%= ENV["STATEMENT_TIMEOUT"] || "5s" %>
```

```suggestion
    statement_timeout: <%= ENV["STATEMENT_TIMEOUT"] || "5s" %>
    idle_in_transaction_session_timeout: <%= ENV["IDLE_IN_TRANSACTION_SESSION_TIMEOUT"] || "90s" %>
```

Can we also add `IDLE_IN_TRANSACTION_SESSION_TIMEOUT` as that that can prevent vacuum from occuring.

---

Discussed irl, not quite needed right now